source/install/desktop.rst

```diff
@@ -57,7 +57,7 @@ The following Group Policies are available:
 macOS 10.9+
 -----------
 
-1. Download the `latest version of the Mattermost desktop app <https://releases.mattermost.com/desktop/4.6.2/mattermost-desktop-4.6.2-mac.dmg>`__.
+1. Download the `latest version of the Mattermost desktop app <https://releases.mattermost.com/desktop/4.7.0/mattermost-desktop-4.7.0-mac.dmg>`__.
```

Comment:
There is now the m1 DMG file for the new apple architecture. We should add that now as it will be replacing the Intel ones in the future. It should have a beta tag

Reply:
Added!